src/service.ts

```diff
@@ -84,6 +85,7 @@ export class Service {
   authClient: GoogleAuth;
   private getCredentials: {};
   readonly apiEndpoint: string;
+  timeout: number;
```

Comment:
Wouldn't this be optional?  Or are we setting it to a default value?

Reply:
I am referencing this [below](https://github.com/googleapis/nodejs-common/pull/455#discussion_r308340988)